src/components/QRShare/index.tsx

```diff
@@ -16,7 +16,7 @@ function QRShare({url, title, subtitle, logo, logoRatio, logoMarginRatio}: QRSha
     const styles = useThemeStyles();
     const theme = useTheme();
 
-    const [qrCodeSize, setQrCodeSize] = useState(1);
+    const [qrCodeSize, setQrCodeSize] = useState(0);
```

Comment:
Why was this changed from 1 to 0? Just wondering since this caused a regression here: https://github.com/Expensify/App/issues/40086

Reply:
IIRC having it set as 0 caused the image to consist of only a couple of pixels due to layout events order.